pkg/core/core.go

```diff
@@ -102,7 +102,7 @@ func run(ctx context.Context, logger *common.Logger) error {
 	txPubsub := pubsub.NewPubsub[struct{}]()
 
 	mempl := mempool.NewMempool(logger, config, db.New(pool), cometConfig.Mempool.Size)
-	mempl.CreateValidatorClients()
+	mempl.InitializeValidatorClients()
```

Comment:
why do we do this twice? can we don't?

Reply:
it's a little ugly but it's so if a node already synced it's initialize here but if it's not synced or a new node it'll get synced later down below